libpod/image/utils.go

```diff
@@ -86,3 +91,10 @@ func getPolicyContext(ctx *types.SystemContext) (*signature.PolicyContext, error
 func hasTransport(image string) bool {
 	return strings.Contains(image, "://")
 }
+
+// CreateStdoutWriter creates an io.Writer bound with os.stdoout
+func CreateStdoutWriter() io.Writer {
```

Comment:
This seems really unnecessary - Go should not require a type cast here?
Also, we're trying to standardize on os.Stderr for progress bars

Reply:
removed